package.json

```diff
@@ -29,12 +29,13 @@
   "homepage": "https://github.com/stampit-org/stamp-specification#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-eslint": "^5.0.0",
+    "babel-eslint": "^6.1.2",
     "babel-polyfill": "^6.7.2",
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.7.2",
-    "check-compose": "^2.0.0",
-    "eslint": "~2.2.0"
+    "check-compose": "^3.1.2",
+    "eslint": "^2.13.1",
```

Comment:
I mean to officially drop support for Node < 4 in a stamp code. It doesn't necessarilly mean any code changes, currently all tests are passing down to 0.10. However eslint v3 has dropped support for these old Node versions so tests would be failing for these.